docs/intro/getting-started.rst

```diff
@@ -2,7 +2,7 @@ Getting Started
 ===============
 
 In this guide we will walk you through setting up your Batavia environment for
-development and testing. We will assume that you have a working Python 3.4,
+development and testing. We will assume that you have an `SSH key <https://help.github.com/articles/generating-an-ssh-key/>`_ set up for your GitHub account, have a working Python 3.4,
```

Comment:
So, I know we hit this error when we were trying to debug your issues, @williln, but a github SSH configuration isn't required by any instruction on this particular page (all `git clone` references use HTTPS). There are two references in `contributing.rst` that I need to clean up, but I'm going to reject this PR based on the above.